frontend/src/app/features/details/summary/SummaryInfo.tsx

```diff
@@ -33,14 +50,58 @@ const SummaryInfo: React.FC<ISummaryInfo> = ({
       console.log('Approve Handler Not Provided');
     });
 
+  const CENTER_OF_BC: LatLngTuple = [53.7267, -127.6476];
+
+  const { data } = useMapSearch_FindSiteBySiteIdQuery({
+    variables: { siteId: siteData.id },
+  });
+
+  const selectedSiteData = data?.findSiteBySiteId.data;
+  const mapRef = useRef<Map>(null);
+  const map_fly_options = { animate: false, duration: 0.2 };
+
+  const MapWithMarker = ({ site }: { site: any }) => {
+    const map = useMap();
+
+    useEffect(() => {
+      if (!site.latdeg || !site.longdeg) return;
+      map.flyTo(
+        {
+          lat: site.latdeg,
+          lng: site.longdeg,
+        },
+        getZoom(map),
+        map_fly_options,
+      );
+    }),
+      [site, map];
```

Comment:
useEffect's dependency array is misplaced and results in build errors. It should be:

```suggestion
    useEffect(() => {
      if (!site.latdeg || !site.longdeg) return;
      map.flyTo(
        {
          lat: site.latdeg,
          lng: site.longdeg,
        },
        getZoom(map),
        map_fly_options,
      );
    }, [site, map]);
```